tiledb/sm/query/readers/sparse_unordered_with_dups_reader.cc

```diff
@@ -1681,117 +1685,35 @@ bool SparseUnorderedWithDupsReader<BitmapType>::process_tiles(
   }
 
   // Read a few attributes at a time.
-  uint64_t buffer_idx = 0;
+  std::optional<std::string> last_field_to_overflow{std::nullopt};
```

Comment:
NOTE TO REVIEWERS: Most of the deleted copy code here was moved as is in copy_tiles.